projects/antimatterDAO/index.js

```diff
@@ -112,26 +121,31 @@ const dualInvestTVL = async (bscBlock) => {
     block: bscBlock,
     chain: "bsc"
   });
+
   sdk.util.sumMultiBalanceOf(balances, usdtValue, true);
   sdk.util.sumMultiBalanceOf(balances, btcValue, true);
-
   return balances;
 };
 
-const stakingTVL = async (ethBlock) => {
+const stakingTVL = async (timestamp, ethBlock) => {
+  const balances = {}
   const { output: staked } = await sdk.api.abi.call({
     target: antimatterStakingContract,
-    abi: abi["TVL"],
+    abi: abi.staking.TVL,
     block: ethBlock,
     chain: "ethereum"
   });
-  return stakingTVL;
+  sdk.util.sumSingleBalance(balances, '0xdac17f958d2ee523a2206206994597c13d831ec7', staked)
+  return  balances
 };
 
-async function bscTVL(block){
+async function bscTVL(timestamp, block){
```

Comment:
the `block` you are using is an ethereum block, what you want is to get the third parameter and extract the chain block from there